### PR TITLE
Added docstring to functions

### PR DESCRIPTION
**Pull request by Omeramsc**
Add docstrings to functions to improve the code's readability.
According to changes asked in #7 comments.

### CODE REVIEW

File: base.py [hunk 1]
```diff
@@ -9,25 +9,30 @@
...
 class GitVendor(metaclass=ABCMeta):
+    """A base, abstract class for the different git vendors."""
     def __init__(self, name, site_url):
         self.name = name
         self.host = site_url
         self.namespaces = {}
 
     @abstractmethod
     def _connect(self):
+        """This function establish a connection to the vendor, in case one is required."""
```
Comment (lioramilbaum) [inline base.py]: No need for the 'This function'

Comment (Omeramsc) [reply base.py]: fixed

File: base.py [hunk 2]
```diff
@@ -128,6 +141,16 @@ def _set_namespaces(self, projects):
...
 
     def get_ccs(self, status='open'):
+        """Pulls and yield CC from the requested Gerrit's namespaces using an SSH command.
+
+              skip: Number of changes to skip.
+              result_cnt: Keeping count of the number of results left to return.
+              cc_content: Outputted JSON of CC from the ssh command.
+              lines: Line-separated JSON copy of cc_content.
+              patch: One gerrit CC.
+
```
Comment (lioramilbaum) [inline base.py]: Remove a blank line

File: main.py [hunk 3]
```diff
@@ -13,6 +13,7 @@
 
 
 def publish(body):
+    """send an email to the selected address with a given body context."""
```
Comment (lioramilbaum) [inline main.py]: s/address/recipients/

**Comment by Omeramsc** [comment]
conflicts fixed, plz re-review. :)

File: base.py (hunk 2 above)
Comment (lioramilbaum) [reply base.py]: Would you like to add comment?

Comment (lioramilbaum) [reply base.py]: next time...

**Pull request merged**